docs/reference/settings/common-defs.asciidoc

```diff
@@ -0,0 +1,167 @@
+tag::ssl-certificate[]
+Specifies the path for the PEM encoded certificate (or certificate chain) that is
+associated with the key.
+//TBD: This setting can be used only if `ssl.key` is set.
```

Comment:
What's the situation with these `//TBD` parts? Are you looking for a review as to whether that content is correct, or trying to decide how to represent them in the docs?

Reply:
@tvernum Sorry for the lack of clarity. These "TBD" pieces are bits of information that only existed in a subset of the places where these definitions were used. For now, I've left them as-is and the TBD is to move that info into the shared definition if indeed they're applicable to all contexts (likely in a subsequent PR).